oshi-core/src/main/java/oshi/software/os/windows/WindowsOperatingSystem.java

```diff
@@ -29,14 +29,7 @@
 import static oshi.util.Memoizer.defaultExpiration;
 import static oshi.util.Memoizer.memoize;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Arrays;
-import java.util.HashSet;
+import java.util.*;
```

Comment:
Thanks for the attempt, but my formatter actually will reverse all this and show the other imports.  I have no idea if it has any actual value to spell them out but for the sake of style/conformity can you leave them as is?

Reply:
yep as .* or typed out? my ide keeps autocorrecting it as .* and failing the star test

Reply:
I meant all individual import statements, no wildcard import.